Lesson Plans/Private/PPL.22 Pilotage and Dead Reckoning.md

```diff
@@ -38,18 +38,33 @@ Ground 1.5 hours, Flight – multiple during dual cross-country flights
 	2. Flight log forms, and updating during flight
 	3. Mental arithmetic - time, speed, distance
 	4. When do we use this in practice?
-3. Complete full navlog
+3.  Important Terminology:
+	1. [[Course]] - intended path over the earth
+	2. [[Heading]] - where the plane's nose is pointing
+	3. [[Track]] - actual path over the ground
+	4. Bearing is the angle from you to some point you want to reach.  Generally, you plan a Course based on the Bearing of your destination or new waypoint.
+	5. [[Magnetic Variation]]
```

Comment:
Might be good to link to [[Compass Errors]] somewhere here. Also, much of this is covered in PPL.19 and PPL.20 currently (the idea being students learn how to plot a course in PPL.19, then how to calculate WCA in PPL.20. by the time they get to PPL.22, it's more about putting together the concepts)